hub/migrations/0037_dataset_is_shadable.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.1.2 on 2023-04-27 14:23
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("hub", "0036_userproperties_agreed_terms"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="dataset",
+            name="is_shadable",
+            field=models.BooleanField(default=True),
+        ),
+    ]
```

Comment:
would be good to add a data migration too, a la b602469, to update the relevant datasets rather than re-running imports

Reply:
And maybe also to do the colours from the previous commit?